project/Dependencies.scala

```diff
@@ -57,9 +57,10 @@ object Dependencies {
   )
 
   lazy val opencensus = zio ++ Seq(
-    Orgs.opencensus % "opencensus-api"               % Versions.opencensus,
-    Orgs.opencensus % "opencensus-impl"              % Versions.opencensus,
-    Orgs.opencensus % "opencensus-contrib-http-util" % Versions.opencensus
+    Orgs.opencensus        % "opencensus-api"               % Versions.opencensus,
+    Orgs.opencensus        % "opencensus-impl"              % Versions.opencensus,
+    Orgs.opencensus        % "opencensus-contrib-http-util" % Versions.opencensus,
+    Orgs.scalaLangModules %% "scala-collection-compat"      % Versions.scalaCollectionCompat
```

Comment:
I assume we need this dep for tests only, so it would be nice to add `% Test`.

Reply:
done